glean-core/ffi/src/lib.rs

```diff
@@ -150,6 +151,26 @@ fn from_raw_string_array_and_string_array(
     }
 }
 
+/// Create a Vec<u32> from a raw C uint64 array.
+///
+/// This will never return an `Error`. However, this is returning
+/// a `Result` for consistency with `from_raw_string_array` and
```

Comment:
We shouldn't return a Result just for consistency with other functions. This only makes us `unwrap` unnecessary below and brings the need to comment why the unwrap is safe.